qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -14,3 +16,19 @@
     test_tags = "evict enlarge"
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
+    variants:
+        - @default:
+            balloon_repeats = 100
+        - with_reboot:
+            pm_test_after_unplug = boot
+            pm_test_after_plug = boot
+        - with_system_reset:
+            pm_test_after_unplug = boot
+            pm_test_after_plug = boot
+            reboot_method = system_reset
+        - with_shutdown_after_unplug:
+            pm_test_after_unplug = shutdown
+            check_img = yes
+        - with_shutdown_after_plug:
+            pm_test_after_plug = shutdown
+            check_img = yes
```

Comment:
I think it's a bit weired here,with_reboot and with_system_reset test cases include unplug and plug actions, but with_shutdown_after_xx  is seperated to two parts.
Why don't change with_shutdown_after_xx  to with_shutdown to keep consistent with reboot and system_reset.

Reply:
If use only one parameter "with_shutdown",then guest will shutdown after balloon device hotplug,then job finished.
shutdown guest after balloon device unpluged will never be ran.

Reply:
Got it ,thanks.
Another question, pm_test_after_unplug and pm_test_after_unplug can be insteaded of pm_test. Why define two parameters?